list-and-titles-of-products.md

```diff
@@ -0,0 +1,37 @@
+The purpose of this PR is to discuss and approve the list and correct titles of all products that we will indicate on the website in the near future.
+
+### Financial area
+
+Ubiquity Dollar 
+
+Ubiquity Card
+
+Ubiquity Payments
```

Comment:
Currently we have it titled as "Ubiquity Rewards" but I want to make it feel like you're logging into you "bank account" (like a normal online bank) with your spendable balance in big numbers.

It would be great if you can work and then see number go up. Perhaps some graphs to show your progress over time and XP so you can feel proud of the hard work you've contributed to the metaverse.